netbox/extras/views.py

```diff
@@ -1126,9 +1127,15 @@ def get(self, request, **kwargs):
         jobs_table = JobTable(
             data=script.jobs.all(),
             orderable=False,
-            user=request.user
+            user=request.user,
+            extra_columns=[(
+                'result', columns.TemplateColumn(
+                    template_code='<a href="{% url "extras:script_result" job_pk=record.pk %}">{{ record.completed }}</a>'
```

Comment:
We've started presenting all dates & times in ISO 8601 format now using the `isoformat()` template filter for consistency.

![screenshot](https://github.com/netbox-community/netbox/assets/13487278/f53e18a8-a8de-4752-941f-963ad8dbc766)